examples/unity-simple-examples/safe_policy.py

```diff
@@ -314,28 +268,4 @@ def step(self, action):
             self.EpisodeCollisions = 0
 
         self.total_steps += 1
-        return observation, reward, done, info
-
-
-if __name__ == '__main__':
-
-    log_dir = "tmp_log/"
-    os.makedirs(log_dir, exist_ok=True)
-
-    file_name = 'DroneDelivery'
-    env_name = "../EnvBuild/" + file_name
-
-    unity_env = UnityEnv(env_name, worker_id=10, use_visual=False)
-    safe_monitor = SafeMonitor(unity_env, log_dir)
-    environment = DummyVecEnv([lambda: safe_monitor])
-
-    if PlotInfoYN:
-        model = PPO2(SafePolicy, environment, verbose=1, learning_rate=learning_rate)
-    else:
-        model = PPO2(SafePolicy, environment, verbose=0, tensorboard_log="./ppo_sb_tensorboard/",
-                     learning_rate=learning_rate)
-
-    model.learn(total_timesteps=1000000, tb_log_name='SafeRL')
-    model.save("unity_SafeRL_model.pkl")
-
-    environment.close()
```

Comment:
It is ok for me to remove the code on `_main_`, but I think it could be great to include the code in a new file ex: `train_agent_safe.py` according the [readme example].(https://github.com/IBM/vsrl-examples/blob/master/examples/unity-simple-examples/README.md#train-a-safe-agent-in-a-unity3d-environment) in the same way that was already done for `train_agent.py`